pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -221,6 +221,27 @@ type ClusterWorkspaceTypeSpec struct {
 	//
 	// +optional
 	LimitAllowedParents *ClusterWorkspaceTypeSelector `json:"limitAllowedParents,omitempty"`
+
+	// defaultAPIBindings are the APIs to bind during initialization of workspaces created from this type.
+	// The APIBinding names will be generated dynamically.
+	//
+	// +optional
+	DefaultAPIBindings []APIExportReference `json:"defaultAPIBindings,omitempty"`
+}
+
+// APIExportReference provides the fields necessary to resolve an APIExport.
+type APIExportReference struct {
+	// workspacePath is the fully-qualified path to the workspace containing the APIExport.
+	//
+	// +required
+	// +kubebuilder:Required
+	WorkspacePath string `json:"workspacePath"`
```

Comment:
Do we call this `workspacePath` anywhere else?

Reply:
I thought `path` was our pattern

Reply:
yeah i'll rename